webview-ui/src/components/chat/CreditLimitError.tsx

```diff
@@ -6,23 +6,30 @@ import { VSCodeButton } from "@vscode/webview-ui-toolkit/react"
 import { useExtensionState } from "@/context/ExtensionStateContext"
 
 import React from "react"
+import { useClineAuth } from "@/context/ClineAuthContext"
 
 interface CreditLimitErrorProps {
 	currentBalance: number
 	totalSpent?: number
 	totalPromotions?: number
 	message: string
-	buyCreditsUrl?: string
+	// buyCreditsUrl?: string
 }
 
 const CreditLimitError: React.FC<CreditLimitErrorProps> = ({
 	currentBalance = 0,
 	totalSpent = 0,
 	totalPromotions = 0,
-	message = "You have run out of credit.",
-	buyCreditsUrl = "https://app.cline.bot/dashboard/account?tab=credits&redirect=true",
+	message = "You have run out of credits.",
+	// buyCreditsUrl = "https://app.cline.bot/dashboard/account?tab=credits&redirect=true",
 }) => {
 	const { uriScheme } = useExtensionState()
+	const { activeOrganization } = useClineAuth()
+
+	const isPersonal = !activeOrganization?.organizationId
+	const buyCreditsUrl = isPersonal
+		? "https://app.cline.bot/dashboard/account?tab=credits&redirect=true"
+		: "https://app.cline.bot/dashboard/organization?tab=credits&redirect=true"
```

Comment:
is checking "activeOrganization" enough because non-owner of an organization will not be able to buy credits, so should we either always send them to personal, or check if they are owner or not first?

Reply:
from slack
> Thats a really good point bee - this is a design decision, we need to show a different ui for a non-admin/owner of an org when out of credits (like ask admin for more?)
> But for now it takes them to our website where hopefully itll be made clear to them